sys-kernel/coreos-firmware/coreos-firmware-99999999.ebuild

```diff
@@ -115,6 +115,10 @@ src_prepare() {
 }
 
 src_install() {
-	insinto /lib/firmware/
-	doins -r *
+	local list=$(find -type f) || die
+
+	if [[ -n ${list} ]] ; then
+		insinto /lib/firmware/
+		doins -r ${list}
+	fi
```

Comment:
At least for amd64 we should emit a warning or possibly even die in this case. Otherwise it is possible to silently miss the fact that `firmware-scan` didn't work correctly which has happened in the past. If arm doesn't include any firmware should we just wrap whatever is depending on coreos-firmware with `amd64? ( sys-kernel/coreos-firmware )` to skip the ebuild entirely?